src/shared_modules/content_manager/src/components/executionContext.hpp

```diff
@@ -29,6 +35,46 @@ const std::string GENERIC_OUTPUT_FOLDER_PATH {std::filesystem::temp_directory_pa
 class ExecutionContext final : public AbstractHandler<std::shared_ptr<UpdaterBaseContext>>
 {
 private:
+    /**
+     * @brief Reads and returns the last offset from the database.
+     *
+     * @param context Updater context configured with the database driver.
+     * @return unsigned int Last offset from the database.
+     */
+    unsigned int getDatabaseOffset(const UpdaterBaseContext& context) const
+    {
+        unsigned int databaseOffset;
+        try
+        {
+            databaseOffset = std::stoi(context.spRocksDB->getLastKeyValue().second.ToString());
+        }
+        catch ([[maybe_unused]] const std::runtime_error& e)
```

Comment:
Nitpick: Use "catch by type"
```suggestion
        catch (const std::runtime_error&)
```

Reply:
Done.